docs/developer.md

```diff
@@ -84,11 +84,16 @@ make catalog-update
 ```sh
 make deploy-olm
 ```
-4. To undeloy the operator with OLM
+5. To undeloy the operator with OLM
 ```sh
 make undeploy-olm
 ```
 
+6. To deploy the operator to Openshift with OLM
+```sh
+make deploy-openshift [IMG=<your-registry>/backstage-operator:tag] [BUNDLE_IMG=<your-registry>/<your-bundle-image>] [CATALOG_IMG=<your-registry>/<your-catalog-image>]
```

Comment:
Wouldn't be simpler to just declare `IMAGE_TAG_BASE`? The repos for operator, bundle and catalog images are derived from that `IMAGE_TAG_BASE` variable..
```suggestion
make deploy-openshift [IMAGE_TAG_BASE=<your-registry>/backstage-operator]
```